src/transformStateWithClones.js

```diff
@@ -7,7 +8,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clones = [];
+  const stateClone = { ...state };
+
+  for (const item of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
better to use clear naming for variables